roles/zabbix_agent/tasks/main.yml

```diff
@@ -46,7 +46,7 @@
     zabbix_agent_listenport: "{{ zabbix_agent_listenport is defined | ternary(zabbix_agent_listenport, zabbix_agent2_listenport) | default(10050) }}"
     zabbix_agent_logfilesize: "{{ zabbix_agent_logfilesize is defined | ternary(zabbix_agent_logfilesize, zabbix_agent2_logfilesize) | default(100) }}"
     zabbix_agent_logtype: "{{ zabbix_agent_logtype is defined | ternary(zabbix_agent_logtype, zabbix_agent2_logtype) | default('file') }}"
-    zabbix_agent_persistentbuffer: "{{ zabbix_agent_persistentbuffer is defined | ternary(zabbix_agent_persistentbuffer, zabbix_agent_persistentbuffer) | default(0) }}"
+    zabbix_agent_enablepersistentbuffer: "{{ zabbix_agent_enablepersistentbuffer is defined | ternary(zabbix_agent_enablepersistentbuffer, zabbix_agent_enablepersistentbuffer) | default(0) }}"
```

Comment:
I think the most "ansible-ism" way to do this would be to put

`zabbix_agent_enablepersistentbuffer: false` in `defaults/main.yml` and then do `"{{ zabbix_agent_enablepersistentbuffer | ternary('1', '0') }}`

Reply:
Ya I get what you're saying.  Right now the stupid thing keeps failing on github and I can't figure out why because when I run the same exact molecule on my box it runs fine.
't

Reply:
OK, ya, I like it, although going make the change across the board for this since there are a number of other variables that are very similar.

Reply:
It seems the zabbix-agent fails to start, could be helpful to see the output. Maybe temporarily put the "start service" in a block, something like

```
- block:
    - service:
        name: zabbix-agent
  rescue:
    - slurp:
        path: /var/log/zabbix/zabbix_agent.log
      register: _res
    - debug:
          var: _res.content | b64decode
    - fail:
```
Just to see what zabbix-agent service is complaining on.

Reply:
Thank you.  I was trying to figure out a way to get that.